### PR TITLE
add delete by id string

### PR DESCRIPTION
**Pull request by wangqj**
add delete by id string

### CODE REVIEW

File: server/resource/v1/kv_resource.go [hunk 1]
```diff
@@ -244,6 +261,40 @@ func (r *KVResource) URLPatterns() []restful.Route {
...
 			Produces: []string{"application/json"},
+		}, {
+			Method:           http.MethodDelete,
+			Path:             "/v1/kv/{ids}",
+			ResourceFuncName: "Delete",
+			FuncDesc:         "delete key by id,seperated by ','",
+			Parameters: []*restful.Parameters{
+				{
+					DataType:  "string",
+					Name:      "X-Domain-Name",
```
Comment (tianxiaoliang) [inline server/resource/v1/kv_resource.go]: use const X-Domain-Name

File: server/resource/v1/kv_resource.go [hunk 2]
```diff
@@ -244,6 +261,40 @@ func (r *KVResource) URLPatterns() []restful.Route {
...
+				{
+					Code:    http.StatusBadRequest,
+					Message: "Failed,check url",
+				},
+				{
+					Code:    http.StatusInternalServerError,
+					Message: "Server error",
+				},
+			},
+			Consumes: []string{goRestful.MIME_JSON},
```
Comment (tianxiaoliang) [inline server/resource/v1/kv_resource.go]: shouldn't add consume and produce, it is a simple delete method

File: server/dao/kv.go [hunk 3]
```diff
@@ -36,7 +36,7 @@ type KV interface {
 	//do not use primitive.ObjectID as return to decouple with mongodb, we can afford perf lost
 	Exist(key, domain string, labels model.Labels) (string, error)
 	DeleteByID(id string) error
-	Delete(key, domain string, labels model.Labels) error
+	Delete(ids string) error
```
Comment (tianxiaoliang) [inline server/dao/kv.go]: use slice, split it in http handler

File: server/resource/v1/kv_resource.go [hunk 4]
```diff
@@ -150,7 +151,23 @@ func (r *KVResource) FindByLabels(context *restful.Context) {
...
+		WriteErrResponse(context, http.StatusInternalServerError, err.Error())
+		return
+	}
+	err = s.Delete(ids)
+	if err != nil {
+		openlogging.Error(fmt.Sprintf("delete ids=%s,err=%s", ids, err.Error()))
+		WriteErrResponse(context, http.StatusInternalServerError, err.Error())
+		return
+	}
+	context.WriteHeader(http.StatusNoContent)
```
[the remark this answers is not in the document]
Comment (wangqj) [reply server/resource/v1/kv_resource.go]: I am following stricter specifications, of course there is no uniform specification here, 
I think as long as our style is the same,Do you think 200 is better?
https://medium.com/@sameira/rest-response-code-for-delete-operation-200-or-204-ee9c72a2dcb2

[the remark this answers is not in the document]
Comment (tianxiaoliang) [reply server/resource/v1/kv_resource.go]: ok. use 204, it makes sense

File: server/resource/v1/kv_resource.go (hunk 1 above)
Comment (wangqj) [reply server/resource/v1/kv_resource.go]: done

File: server/resource/v1/kv_resource.go (hunk 2 above)
Comment (wangqj) [reply server/resource/v1/kv_resource.go]: done

File: server/dao/kv.go (hunk 3 above)
Comment (wangqj) [reply server/dao/kv.go]: done

**Comment by tianxiaoliang** [comment]
LGTM

**Comment by WillemJiang** [comment]
LGTM

**Pull request merged**